joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/CallTests.scala

```diff
@@ -89,7 +89,7 @@ class CallTests extends PhpCode2CpgFixture {
 
     inside(cpg.call.l) { case List(fooCall) =>
       fooCall.name shouldBe "foo"
-      fooCall.methodFullName shouldBe s"<empty>"
+      fooCall.methodFullName shouldBe s"<unresolvedNamespace>\\$$f->foo"
```

Comment:
I like writing strings like this with `"""` so that there is no need to escape anything:

```suggestion
      fooCall.methodFullName shouldBe """<unresolvedNamespace>\$f->foo"""
```